packages/next/src/client/components/react-dev-overlay/ui/components/version-staleness-info/version-staleness-info.tsx

```diff
@@ -8,10 +8,28 @@ export function VersionStalenessInfo({
   versionInfo: VersionInfo
   isTurbopack?: boolean
 }) {
-  const { staleness } = versionInfo
-  let { text, indicatorClass, title } = getStaleness(versionInfo)
+  if (versionInfo.staleness === 'unknown') {
+    // This is only available for `__next_root_layout_missing_tags` error.
+    const versionInfoInit = window.__NEXT_VERSION_INFO_INIT
 
-  const shouldBeLink = staleness.startsWith('stale')
+    if (versionInfoInit) {
+      versionInfo = versionInfoInit
```

Comment:
Is it ok to override the function argument here, or would it be better to preserve an internal copy instead? I think unexpected mutations are not always best-practice.

Also, not exactly sure how safe using the `window.__*` variable is to override with is? I understand we have the typedef, but not sure if that deems this safe to do?